herpetological-conservation-and-biology.csl

```diff
@@ -0,0 +1,223 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" default-locale="en-US" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>Herpetological Conservation and Biology</title>
+    <title-short>HCB</title-short>
+    <id>http://www.zotero.org/styles/herpetological-conservation-and-biology</id>
+    <link href="http://www.zotero.org/styles/herpetological-conservation-and-biology" rel="self"/>
+    <link href="http://www.herpconbio.org/InstructionsForAuthors.pdf" rel="documentation"/>
+    <author>
+      <name>Matthew H. Shirley</name>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="biology"/>
+    <category field="zoology"/>
+    <issn>1931-7603</issn>
+    <updated>2016-03-02T15:22:52</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="author">
+    <names variable="author" suffix=".">
+      <name and="text" delimiter-precedes-last="always" et-al-min="10" et-al-use-first="10" initialize-with="." name-as-sort-order="first"/>
```

Comment:
set the `et-al` below in citation, so you only have to do this once. Also, you want the `et-al-min` to always be at least one bigger than the `et-al-use-first`. Having them at the same number is non-sensical (it'd imply using et al for an article with 10 authors and printing the first ten authors)